src/siren.ts

```diff
@@ -81,8 +81,7 @@ export type RelValue =
       | "via"
       | "webmention"
       | "working-copy"
-      | "working-copy-of"
```

Comment:
:question: Did you mean to do this?

Reply:
`yarn run fix` did this automatically. I reverted the change but need to investigate why I get different results than github-actions.